dolt_integrations/metaflow/examples/hospital_bounty/blog_post.md

```diff
@@ -0,0 +1,270 @@
+| q49l0kgnbbbgkt3imjd57tslbi2iges8 | Updating hospitals with changes from hospitals.csv         |
+| gstcq5loi9ieqdv1elrljab9hcgr090p | Updating hospitals with changes from hospitals.csv         |
+| te6spcqtjk0scose2c45f9t7tpcrt69c | Added hospital WellSpan Surgery & Rehabilitation Hospital. |
+| bjg3b5lua8omadcl5nr6o7v0nphliqpu | Added hospital WellSpan York Hospital.                     |
+| t4js1g5mfvgikqlmqa238it26mg94g5i | Added hospital Children's of Alabama.                      |
+| jsan7p4iad61cjmeti858ebcl4s86vda | Added hospital McLaren Lapeer Region.                      |
++----------------------------------+------------------------------------------------------------+
+```
+
+Suppose we'd like to run our pipeline with input data as of commit `gstcq5loi9ieqdv1elrljab9hcgr090p`. That's easy enough, first let's name the commit with a branch:
+```
+$ dolt branch metaflow-backtest gstcq5loi9ieqdv1elrljab9hcgr090p
+```
+
+Now let's kick off recomputing the medians. Since we are recomputing our medians from a historical version of the raw pricing data we will write them to a separate experimentation branch:
+```
+$ poetry run python3 hospital_procedure_price_state_medians.py run \ 
+    --hospital-price-db path/to/hospital-price-transparency \ 
+    --hospital-price-db-branch metaflow-backtest \
+    --hospital-price-analysis-db path/to/hospital-price-analysis \
+    --hospital-price-analysis-db-branch metaflow-backtest
+```
+
+And the variances can be run similarly, again using the experimentation branch:
+```
+$ poetry run python3 hospital_procedure_price_variance_by_state.py run \ 
+    --hospital-price-analysis-db path/to/hospital-price-analysis \
+    --hospital-price-analysis-db-branch metaflow-backtest
+```
+
+We can now query the results directly from Python:
+```python
+from metaflow import Flow
+from dolt_integrations.metaflow import DoltDT
+dolt = DoltDT(run=Flow('HospitalProcedurePriceVarianceByState').latest_successful_run)
+df = dolt.read('variance_by_procedure')
+print(df)
+```
+
+Or we can use Dolt SQL to query the results directly:
+```
+$ cd path/to/hospital-price-analysis
+$ dolt sql
+# Welcome to the DoltSQL shell.
+# Statements must be terminated with ';'.
+# "exit" or "quit" (or Ctrl-D) to exit.
+hospital_price_analysis> 
+```
+
+In this section we saw how storing flow inputs in Dolt makes back-testing straightforward. Dolt's commit graph makes it straight forward to specify a historical database state. 
+
+### Reproducibility
+Our pipeline contains two steps, one computes state procedure price medians, and the second computes procedure price variances across states. Suppose now that we would like to tweak the way we compute variances. We might like to exclude some outliers, or invalid procedure codes. In a production setting we might own the variances computation but not the medians computation, and have stricter criteria for excluding invalid data. Let's update our variances job and then recompute using a fixed.
```

Comment:
variance's and median's

Reply:
I don't think this is actually possessive since those are plurals